values-hr

```diff
@@ -0,0 +1,181 @@
+            <string name="label_close">Zatvori</string>
+            <string name="label_adblock_detected">Otkriveno je blokiranje oglasa</string>
+            <string name="label_permission_denied">Dopuštenje je odbijeno</string>
+            <string name="permission_granted">Dopuštenje je dozvoljeno! Hvala!</string>
+            <string name="label_grant_permission">Dozvoli dopuštenje</string>
+            <string name="navigation_drawer_open">Otvori navigacijski izbornik</string>
+            <string name="navigation_drawer_close">Zatvori navigacijski izbornik</string>
+            <string name="label_accept">Prihvati</string>
+            <string name="label_accepted">Prihvaćeno</string>
+            <string name="label_decline">Odbij</string>
+            <string name="label_delete">Izbriši</string>
+            <string name="label_delete_warning">Želite li izbrisati %1$s?</string>
+            <string name="dont_show_again">Ne prikazuj ponovno</string>
+            <string name="title_optional_app_update">Dostupno je ažuriranje aplikacije</string>
+            <string name="ms_prev">Prethodno</string>
+            <string name="ms_next">Dalje</string>
+            <string name="ms_end">Završi</string>
+        <!--FEEDBACK-->
+            <string name="label_now">Sada</string>
+            <string name="label_later">Kasnije</string>
+            <string name="label_never">Nikad</string>
+            <string name="label_rate">Ocijeni</string>
+            <string name="title_rate_us">Ocijenite ovu aplikaciju.</string>
+            <string name="message_rate_us">Nadamo se da uživate u korištenju ovom aplikacijom. Treba vam manje od minute da nas ocijenite ili pošaljete povratne informacije. To će pomoći i drugima da otkriju ovu aplikaciju!</string>
+            <string name="label_supportproject">Podrška</string> <!--verb-->
+            <string name="message_supportproject">Ako vam se sviđaju OpenGApps paketi, razmislite o tome da podržite ovaj projekt. Razvoj paketa potpuno je otvorenog koda i svi imaju koristi od truda razvojnih programera!</string>
+        <!--EXPLANATION-->
+            <string name="explanation_permission_denied">Da bi sve ispravno radilo morate OpenGAppsu dozvoliti dopuštenje za pristup vanjskoj pohrani. Ima smisla, zar ne?</string>
+            <string name="explanation_adblock_detected">Otkriveno je blokiranje oglasa!? Oglasi znaju biti frustrirajući, ali oni održavaju projekt i njegove programere živima! Molimo vas da razmislite o onemogućavanju blokiranja oglasa, ili možda donaciji</string>
+            <string name="title_webview_not_installed">Ups! Nije instaliran nijedan pružatelj web-pregleda!?</string>
+            <string name="message_webview_not_installed">Čini se da ste deinstalirali \"zadani web-pregled\", a vaš ROM ne podržava \"Googleov web-pregled\". Ponovno instalirajte ROM i pitajte programera ROM-a da doda podršku za alternativne pružatelje web-pregleda.</string>
```

Comment:
It might be best not to translate "Stock WebView" and the "Google WebView" and the term WebView Provider, because they are technical terms, and would allow the user to search for the issue on the internet.

In i.e. the Chinese translation it is done with using those English terms, and the Chinese name/explanation behind it in parenthesis. Would you be able to do something similar?